swarm/src/behaviour.rs

```diff
@@ -721,3 +850,182 @@ impl Default for CloseConnection {
         CloseConnection::All
     }
 }
+
+// TODO: Rename to FromSwarm
```

Comment:
Should we pool the rename together with the release of this change as well?

Reply:
We could also name it `FromTransport` maybe? I am beginning to more and more take on the notion of `libp2p-swarm` being a kind of "runtime" that actually doesn't do anything by itself but passes data between other components so in a way, the swarm itself would never produce an event.

Reply:
In favor of naming it `FromSwarm`. `FromTransport` imo could be confusing because its not an event emitted by a transport **implementation** (e.g. in the relay protocol we do have events that are actually produced by the relay client transport). Also, some events are still caused by the swarm and not the transport, e.g. a `DialFailure` because of swarm connection limits.
That being said, I do generally like the conception of swarm being some kind of "runtime".